src/etc/inc/auth.inc

```diff
@@ -488,7 +488,7 @@ function local_user_del($user)
 
 function local_user_set_password(&$user, $password)
 {
-    $user['password'] = crypt($password, '$6$');
+    $user['password'] = generate_password_hash($password, 10);
```

Comment:
I've tested this with my APU1D, and if I had chosen 12, then password verification would take about half a second. Might be too much for a default for everyone.